### PR TITLE
Plasmamen organs GET sprites!

### PR DESCRIPTION
**Pull request by ghost**
:cl: Onule!
add: Plasmaman liver and stomach crystals!
/:cl:

Oh my lord, they didn't even have sprites... ech.

### CODE REVIEW

**Comment by Armhulen** [comment]
@icondiffbot

**Comment by KorPhaeron** [comment]
fantastic

**Review by WJohn — DISMISSED** [review]

File: code/modules/surgery/organs/organ_internal.dm [hunk 1]
```diff
@@ -114,7 +114,7 @@
 			breathes = FALSE
 		if(NOBLOOD in dna.species.species_traits)
 			blooded = FALSE
-		var/has_liver = (!(NOLIVER in dna.species.species_traits))
+		var/has_liver = (!(NOLIVER in dna.species.species_traits)) //
```
Comment (KorPhaeron) [inline code/modules/surgery/organs/organ_internal.dm]: Why

**Review by imtakingabreakdontatme — DISMISSED** [review]

**Comment by Tacolizard** [comment]
they had sprites originally there must have been an issue. maybe I forgot to merge them or something

**Pull request merged**